src/Toolkit/V.php

```diff
@@ -239,13 +239,45 @@ public static function __callStatic(string $method, array $arguments): bool
     },
 
     /**
-     * Checks for a valid date
-     */
-    'date' => function ($value): bool {
-        $date = date_parse($value);
-        return $date !== false &&
-                $date['error_count'] === 0 &&
-                $date['warning_count'] === 0;
+     * Checks for a valid date and is able to compare
+     * dates against each other.
+     *
+     * Pass only the first argument to check for a valid date.
+     * Pass an operator as second argument and another date as
+     * third argument to compare them.
+     */
+    'date' => function ($value, string $operator = null, $test = null): bool {
+        $args = func_get_args();
+
+        // simple date validation
+        if (count($args) === 1) {
+            $date = date_parse($value);
+            return $date !== false &&
+                    $date['error_count'] === 0 &&
+                    $date['warning_count'] === 0;
+        }
+
+        $value = strtotime($value);
+        $test  = strtotime($test);
+
+        if (is_int($value) !== true || is_int($test) !== true) {
+            return false;
+        }
+
+        switch ($operator) {
+            case '!=':
+                return $value != $test;
+            case '<':
+                return $value < $test;
+            case '>':
+                return $value > $test;
+            case '<=':
+                return $value <= $test;
+            case '>=':
+                return $value >= $test;
+            default:
+                return $value == $test;
```

Comment:
I wouldn't use the `switch` default branch to be honest. Like in Nico's implementation, the validator should throw an Exception if the operator is invalid. Swallowing such an error and just doing an arbitrary comparison is going to lead to bugs that are hard to debug.

Reply:
You are right. I fixed it.